dashboard/app/controllers/plc/content_creator_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Plc::ContentCreatorController < ApplicationController
-  before_filter :require_levelbuilder_mode
-
   def show_courses_and_modules
+    authorize! :read, Plc::Course
```

Comment:
The description says restricted to users who can *edit* courses. That should be authorize! :manage. This is read access. Either the description is wrong or this is too permissive.